src/Components/NumberInputTextField/NumberInputTextField.tsx

```diff
@@ -0,0 +1,26 @@
+import TextField from '@mui/material/TextField'
+import { ChangeEvent } from 'react'
+
+type NumberInputTextFieldProps = {
+    label: string
+    value: number
+    onChangeEvent: (
```

Comment:
You can simplify this typing by referencing original mui's TextField's prop like this:
```
import TextField, { TextFieldProps } from '@mui/material/TextField'


type NumberInputTextFieldProps = TextFieldProps
```